src/main/controllers/areaOfLaw/FamilyAreaOfLawController.ts

```diff
@@ -0,0 +1,48 @@
+import { FactRequest } from '../../interfaces/FactRequest';
+import { hasProperty, isObjectEmpty } from '../../utils/validation';
+import { PageData } from '../../interfaces/PageData';
+import { Response } from 'express';
+import { FactApi } from '../../utils/FactApi';
+import { FamilyAreaOfLawData } from '../../interfaces/FamilyAreaOfLawData';
+import autobind from 'autobind-decorator';
+
+@autobind
+export class FamilyAreaOfLawController {
```

Comment:
I think you can make one controller for all areas of law: to handle `/services/{areaOfLaw}/service-areas`